scipy/stats/_qmc.py

```diff
@@ -401,7 +446,7 @@ def n_primes(n):
               677, 683, 691, 701, 709, 719, 727, 733, 739, 743, 751, 757, 761,
               769, 773, 787, 797, 809, 811, 821, 823, 827, 829, 839, 853, 857,
               859, 863, 877, 881, 883, 887, 907, 911, 919, 929, 937, 941, 947,
-              953, 967, 971, 977, 983, 991, 997][:n]
+              953, 967, 971, 977, 983, 991, 997])[:n]
```

Comment:
@BvB93 Do you know how to to type `n` ? 
For now, we have the following error :
```bash
scipy/stats/_qmc.py:449: error: Slice index must be an integer or None  [misc]
scipy/stats/_qmc.py:454: error: Slice index must be an integer or None  [misc]
```
because `builtin.slice` is not accepting other types than `int` or `None` (see python/typing#159)
Apparently (https://github.com/python/mypy/issues/2410#issuecomment-260159393), NumPy `slice` method accepts other types. Do you know how to correct this ?

Reply:
Given the current mypy issue, I'm afraid the only way to deal with this false positive is to add a `# type: ignore[misc]` comment.